apps/sms/js/smil.js

```diff
@@ -67,11 +67,15 @@ function SMIL_generateSlides(data, slide, slideIndex) {
   if (slide.blob) {
     blobType = Utils.typeFromMimeType(slide.blob.type);
     if (blobType) {
+      var region = 'region="Image"';
```

Comment:
nit: How about having a comment about we don't set region attribute for vcard attachment ?